other/amo.md

```diff
@@ -50,3 +50,13 @@ Risks for the protocol are reduced because it is expected that the partner DAO i
 So far, Angle has worked with Ondo to offer such services, as presented in [this](https://gov.angle.money/t/proposal-angle-ondo-liquidity-as-a-service-program/320) governance discussion.
 
 The first liquidity as a service operation has been approved with Paladin as detailed [here](https://gov.angle.money/t/liquidity-as-a-service-partnership-with-paladin/322), to provide 500,000 agEUR as liquidity to PAL through Ondo contracts.
+
+### Protocol-owned Liquidity
+
+Angle is using some of its governance surplus in USDC coupled with an agEUR AMO to provide liquidity on Optimism and Arbitrum agEUR/USDC pools.
+
+One pool on each of the two networks has been seeded with 200,000 USDC from the surplus, and 200,000 agEUR minted through AMOs, for a total of 400,000 new agEUR coming from AMOs.
```

Comment:
Not exactly 400k new agEUR: we mint what's needed to match with USDC

Reply:
Let's try to be generic and say that protocol uses liquidity and then dropdown list with all the things that have been deployed so far